chipsec/logger.py

```diff
@@ -194,121 +247,135 @@ def disable(self):
         self.close()
         self.Results.flush_testcases()
 
-    ######################################################################
-    # Logging functions
-    ######################################################################
-
     def flush(self):
         sys.stdout.flush()
         if self.LOG_TO_FILE and self.logfile is not None:
             # flush should work with new python logging
             try:
-                self.rootLogger.removeHandler(self.logfile)
+                self.chipsecLogger.removeHandler(self.logfile)
                 self.logfile.flush()
-                self.rootLogger.addHandler(self.logfile)
+                self.chipsecLogger.addHandler(self.logfile)
             except Exception:
                 self.disable()
 
     def set_always_flush(self, val):
         self.ALWAYS_FLUSH = val
 
-    def _log(self, text, level=pyLogging.INFO, color=None):
+    # -------------------------------------------------------
+    # These logger methods are deprecated and will be removed
+    # -------------------------------------------------------
+
+    def _log(self, text, level=logging.INFO, color=None):
         """Sends plain text to logging."""
         if self.Results.get_current() is not None:
             self.Results.get_current().add_output(text)
         try:
-            self.rootLogger.log(level, text, color)
+            self.chipsecLogger.log(level, text, color)
             if self.ALWAYS_FLUSH:
                 self.flush()
         except BaseException:
             print(text)
 
-    def log(self, text):
-        """Plain Log message"""
-        self._log(text, pyLogging.INFO, "WHITE")
-
-    def log_verbose(self, text):
+    def log_verbose(self, text):  # Use log('text', self.logger.VERBOSE)
         """Logs a Verbose message"""
```

Comment:
Can log_hal, log_verbose, and log_debug be left off the deprecated function list.  I see value in calling them directly.

Reply:
The idea is to only have one log function. Do you not see the same value between log_hal() and log(level.HAL)? What makes those better options?

Reply:
Exactly. We can discuss details later

Reply:
When would be a good time to discuss?

Reply:
let's set up a meeting